R/r.R

```diff
@@ -88,6 +88,39 @@ use_test <- function(name = NULL, open = rlang::is_interactive()) {
   invisible(TRUE)
 }
 
+#' Edit the snapshot file associated with an R or test file
+#'
+#' This function opens the snapshot file associated with an R or test file, if
+#' one exists.
+#'
+#' @inheritParams use_r
+#' @param variant An optional subdirectory within the `_snaps/` directory.
```

Comment:
I think you can drop the variant argument; it's not important for this use case.